frontend/src/views/WheresMyGene/components/InfoPanel/components/Legend/index.tsx

```diff
@@ -61,6 +65,8 @@ export default memo(function Legend({
             setEchartsRendererMode={setEchartsRendererMode}
             allChartProps={allChartProps}
             availableFilters={availableFilters}
+            tissues={tissues || []}
```

Comment:
passing `[]` will be a new array every time and failing reference equality check `===`, so let's use `EMPTY_ARRAY` instead!